src/runtime/methodbinder.cs

```diff
@@ -211,53 +294,45 @@ internal static int ArgPrecedence(Type t)
                 case TypeCode.UInt64:
                     return 10;
 
-                case TypeCode.UInt32:
+                case TypeCode.Int64:
```

Comment:
These changes to the argument type priority are breaking.
We can consider them for Python.NET 3.0+, but a rationale and corresponding documentation is required.